misc/python/materialize/xcompile.py

```diff
@@ -91,7 +91,7 @@ def cargo(
         extra_env = {
             f"CMAKE_SYSTEM_NAME": "Linux",
             f"CARGO_TARGET_{_target_env}_LINKER": f"{_target}-cc",
-            f"CARGO_TARGET_DIR": ROOT / "target-xcompile",
+            f"CARGO_TARGET_DIR": str(ROOT / "target-xcompile"),
```

Comment:
Ah yeah, this is in a `sys.platform == "darwin"` block so it goes unnoticed by CI. Thanks!